framework/src/bluetooth/adaptation/bt-adatation-adapter.c

```diff
@@ -15,6 +15,7 @@
  *
  */
 
+#include <tinyara/config.h>
```

Comment:
There is no CONFIG_XX definition. Could you let me know the reason of this commit?

Reply:
This is template code. So, CONFIG_XX definition can be added in later.
and, for consistency, I added this line all of bluetooth files.